app/src/main/java/org/listenbrainz/android/ui/theme/Color.kt

```diff
@@ -30,3 +30,4 @@ val yimYellow = Color(0xFFFECB49)
 val yimRed = Color(0xFFFE0E25)
 val yimWhite = Color(0xFFFEFEFE)
 val yimOffWhite = Color(0xFFF4F4F4)
+
```

Comment:
Avoid mutating files which don't change anything.

Reply:
Okay okay, I will remember this. But please give me one answer whether I reached the desire behaviour or not.

Because, These commented code is only for my uses